src/backend/oneapi/kernel/tile.hpp

```diff
@@ -0,0 +1,116 @@
+/*******************************************************
+ * Copyright (c) 2022, ArrayFire
+ * All rights reserved.
+ *
+ * This file is distributed under 3-clause BSD license.
+ * The complete license agreement can be obtained at:
+ * http://arrayfire.com/licenses/BSD-3-Clause
+ ********************************************************/
+
+#pragma once
+
+#include <Param.hpp>
+#include <common/dispatch.hpp>
+#include <common/kernel_cache.hpp>
+#include <debug_oneapi.hpp>
+
+#include <string>
+#include <vector>
+
+namespace arrayfire {
+namespace oneapi {
+namespace kernel {
+
+template<typename T>
+using local_accessor = sycl::accessor<T, 1, sycl::access::mode::read_write,
```

Comment:
Test without, this should now be part of the 2020 spec as sycl::local_accessor

Reply:
turns out intel compiler doesn't have sycl::local_accessor. we might be able to adopt this later